src/apache/conf/httpd.conf

```diff
@@ -48,6 +48,22 @@ PidFile "${APACHE_PIDFILE}"
 # LoadModule foo_module modules/mod_foo.so
 #
 
+<IfDefine EnableCompression>
+  #
+  # This configuration is based on what can be read here:
+  # https://github.com/nextcloud/documentation/blob/master/admin_manual/installation/nginx.rst
+  # https://blog.cloudflare.com/results-experimenting-brotli/
+  # https://blogs.akamai.com/2016/02/understanding-brotlis-potential.html
+  #
+  Header append Vary User-Agent
+  BrotliCompressionQuality 1
+  AddOutputFilterByType BROTLI_COMPRESS application/atom+xml application/javascript application/json application/ld+json application/manifest+json application/rss+xml application/vnd.geo+json application/vnd.ms-fontobject application/x-font-ttf application/x-web-app-manifest+json application/xhtml+xml application/xml font/opentype image/bmp image/svg+xml image/x-icon text/cache-manifest text/css text/plain text/vcard text/vnd.rim.location.xloc text/vtt text/x-component text/x-cross-domain-policy
+</IfDefine>
+
+
+#<IfModule deflate_module>
```

Comment:
Left over from testing?

Reply:
Well, I wanted to document somewhere why I hardcoded the compression level, but I can delete it and write it on the PR or the commit message if you want.

Reply:
Yes